buildtest/cli/help.py

```diff
@@ -25,6 +25,10 @@ def print_build_help():
         "buildtest build -t pass -t python",
         "Build buildspecs by tagname 'pass' and 'python'",
     )
+    table.add_row(
+        "buildtest build -t pass,fail -t network",
```

Comment:
i think you should remove the `-t network` since we do the appending example above. This will match the description for this row entry